upload.linq

```diff
@@ -42,6 +42,9 @@ let toLower (s : string) = s.ToLower()
 let getDirectory dir = 
     (new DirectoryInfo(dir)).EnumerateFiles("*.*", SearchOption.AllDirectories)
 
+let pFilter p (s : seq<'T>) = 
+    ParallelEnumerable.Where(s.AsParallel(), Func<_,_>(p))
```

Comment:
should we add `.WithDegreeOfParallelism(6)` 

(6 is seemingly the number of concurrent uploads that iBroadcast allows)